### PR TITLE
remove small objects

### PR DESCRIPTION
**Pull request by rijobro**
### Description
Post transform to remove small objects.

Merge https://github.com/Project-MONAI/DocImages/pull/13 when happy.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] New tests added to cover the changes.

### CODE REVIEW

**Review by ericspod — APPROVED** [review]

File: monai/transforms/utils.py [hunk 1]
```diff
@@ -979,6 +981,51 @@ def get_largest_connected_component_mask(img: NdarrayTensor, connectivity: Optio
...
+    if not independent_channels:
+        img_np = img_np > 0
+    else:
+        # if binary, convert to boolean, else int
+        img_np = img_np.astype(bool if img_np.max() <= 1 else np.int32)
+
+    out = morphology.remove_small_objects(img_np, min_size, connectivity)
+    # convert back by multiplying
+    if not independent_channels:
+        out = img * out
```
Comment (wyli) [inline monai/transforms/utils.py:1025]: error from this 
```
[2022-08-25T16:29:24.326Z] ======================================================================
[2022-08-25T16:29:24.326Z] ERROR: test_remove_small_objects_dict_34 (tests.test_remove_small_objects.TestRemoveSmallObjects)
[2022-08-25T16:29:24.326Z] ----------------------------------------------------------------------
[2022-08-25T16:29:24.326Z] Traceback (most recent call last):
[2022-08-25T16:29:24.326Z]   File "/usr/local/lib/python3.7/dist-packages/parameterized/parameterized.py", line 533, in standalone_func
[2022-08-25T16:29:24.326Z]     return func(*(a + p.args), **p.kwargs)
[2022-08-25T16:29:24.326Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_remove_small_objects.py", line 67, in test_remove_small_objects_dict
[2022-08-25T16:29:24.326Z]     lbl_clean = RemoveSmallObjectsd("lbl", **params)({"lbl": lbl})["lbl"]
[2022-08-25T16:29:24.326Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/post/dictionary.py", line 296, in __call__
[2022-08-25T16:29:24.326Z]     d[key] = self.converter(d[key])
[2022-08-25T16:29:24.326Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/post/array.py", line 423, in __call__
[2022-08-25T16:29:24.326Z]     return remove_small_objects(img, self.min_size, self.connectivity, self.independent_channels)
[2022-08-25T16:29:24.326Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utils.py", line 1025, in remove_small_objects
[2022-08-25T16:29:24.326Z]     out = img * out
[2022-08-25T16:29:24.326Z]   File "/usr/local/lib/python3.7/dist-packages/torch/tensor.py", line 621, in __array__
[2022-08-25T16:29:24.326Z]     return self.numpy()
[2022-08-25T16:29:24.326Z] TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
[2022-08-25T16:29:24.326Z] 
[2022-08-25T16:29:24.326Z] ----------------------------------------------------------------------
```

Comment (ericspod) [inline monai/transforms/utils.py:1025]:
```suggestion
        out = img_np * out
```
?

**Pull request merged**